examples/todos/src/containers/AddTodo.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import AddTodo from './AddTodo';
+import {shallow} from 'enzyme'
+import configureStore from 'redux-mock-store';
+
+const setup = () => {
+  const store = configureStore()({ todos: [] });
+  const wrapper = shallow(<AddTodo store={store} />);
+
+  return {
+    store,
+    wrapper
+  };
+};
+
+describe('Test AddTodo',  () => {
+    it('should match snapshot', () => {
```

Comment:
For submitting action, maybe we missing test case here.
- What if `value` is empty when we click on submit button? We should not do anything: dispatch value or set update state, right?